can-route-pushstate.md

```diff
@@ -8,40 +8,62 @@
 
 @type {RoutePushstate}
 
-__can-route-pushstate__ exports a `RoutePushstate` constructor function that configure [can-route] to use
```

Comment:
These 2 sentences say it exports different things.

It exports a constructor function.  That constructor function can be used to create obserables that can be used with `can-route`.